### PR TITLE
6531 na option

### PR DESCRIPTION
**Pull request by emitche**

### CODE REVIEW

File: app/views/admin/loans/questionnaires/_questionnaire_group.html.slim [hunk 1]
```diff
@@ -57,62 +57,70 @@
...
+                question: question, document: response.linked_document
 
-            - if value.blank?
-              .view-element.answer.blank = t("loan.no_answer")
+            .not-applicable
+              - if response.not_applicable?
+                .view-element.answer = t("loan.not_applicable")
+
+              .form-element
+                /= f.input_field "#{question.attribute_sym}[not_applicable]", as: :boolean
```
Comment (emitche) [inline app/views/admin/loans/questionnaires/_questionnaire_group.html.slim]: I can use help here. I'd like to use Simple Form for this part, but am getting an error. Any ideas welcome. Thank you!

File: app/views/admin/loans/questionnaires/_questionnaire_group.html.slim [hunk 2]
```diff
@@ -57,62 +57,76 @@
...
+              .form-element
+                / The boolean field above uses a select box so we need a different treatment here.
+                / However it makes sense to maintain the same underlying boolean values (yes/no) in the
+                / response data. It is easiest to do a manual checkbox, plus a hidden tag so that if
+                / the box is unchecked, the submitted value will be 'no'. This is a standard Rails trick:
+                / https://apidock.com/rails/ActionView/Helpers/FormHelper/check_box
+                = hidden_field_tag :"loan_response_set[#{question.attribute_sym}][not_applicable]", "no"
+                = check_box_tag :"loan_response_set[#{question.attribute_sym}][not_applicable]",
+                  "yes", response.not_applicable?
+                = label_tag "loan_response_set[#{question.attribute_sym}][not_applicable]", t("loan.not_applicable")
```
Comment (hooverlunch) [inline app/views/admin/loans/questionnaires/_questionnaire_group.html.slim]: @emitche you can see my changes here. Let me know if you have questions.

File: .travis.yml [hunk 3]
```diff
@@ -10,6 +10,7 @@ addons:
 services:
   - postgresql
 before_install: # Install PhantomJS 2.1.1 or some of our specs fail with default (v1.9.8 as of 3/2/2017)
+  - gem update bundler --no-rdoc --no-ri
```
[the remark this answers is not in the document]
Comment (hooverlunch) [reply .travis.yml]: Normally I'd agree but I put them in here in order to get CI working again, without which it's harder to know if the code is ready for merge.

[the remark this answers is not in the document]
Comment (hooverlunch) [reply .travis.yml]: If you like I can remove them, just let me know.

**Comment by emitche** [comment]
Failed test expected and not related to these changes.

File: .travis.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (emitche) [reply .travis.yml]: Let's keep things simple and just keep it in this branch. Thank you! Makes sense.

**Pull request merged**